layouts/default.vue

```diff
@@ -3,9 +3,9 @@ import { SpeedInsights } from '@vercel/speed-insights/nuxt';
 </script>
 
 <template>
-  <div class="flex h-screen flex-col justify-between px-10 py-5">
+  <div class="flex h-screen flex-col justify-between">
     <Header />
-    <main class="h-full py-10">
+    <main class="m-auto min-h-[100%] max-w-[1440px]">
```

Comment:
We can use 'min-h-full' instead of min-h-[100%]. And I think would be easier to define 1440 px as a screen size, since we'll be using it everywhere.